From: Markus Koschany <apo@gambaru.de>
Date: Sat, 15 Dec 2012 22:29:35 +0000
Subject: fix format is not a string literal error

Xarchiver is compiled with Werror=format-security. All warnings are treated as
errors. This patch prevents a FTBFS.
---
 src/window.c |    6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/window.c b/src/window.c
index f332e2c..abeec1d 100644
--- a/src/window.c
+++ b/src/window.c
@@ -231,9 +231,9 @@ int xa_show_message_dialog (GtkWindow *window,int mode,int type,int button,const
 {
 	int response;
 
-	dialog = gtk_message_dialog_new (window,mode,type,button,message1);
+	dialog = gtk_message_dialog_new (window,mode,type,button,"%s",message1);
 	gtk_dialog_set_default_response (GTK_DIALOG (dialog),GTK_RESPONSE_NO);
-	gtk_message_dialog_format_secondary_text (GTK_MESSAGE_DIALOG (dialog),message2);
+	gtk_message_dialog_format_secondary_text (GTK_MESSAGE_DIALOG (dialog),"%s",message2);
 	response = gtk_dialog_run (GTK_DIALOG (dialog));
 	gtk_widget_destroy (GTK_WIDGET (dialog));
 	return response;
@@ -509,7 +509,7 @@ void xa_list_archive (GtkMenuItem *menuitem,gpointer data)
 			g_fprintf (stream,_("Comment:\n"));
 			if (bp)
 				g_fprintf(stream,"</b><pre>");
-			g_fprintf (stream,archive[idx]->comment->str);
+			g_fprintf (stream,"%s",archive[idx]->comment->str);
 			if (bp)
 				g_fprintf(stream,"</pre>");
 			g_fprintf (stream,"\n");
